modules/exploits/linux/http/f5_icontrol_rce.rb

```diff
@@ -0,0 +1,81 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Undisclosed requests may bypass iControl REST authentication',
+        'Description' => %q{
+          On F5 BIG-IP 16.1.x versions prior to 16.1.2.2, 15.1.x versions prior to 15.1.5.1,
+          14.1.x versions prior to 14.1.4.6, 13.1.x versions prior to 13.1.5, and all 12.1.x
+          and 11.6.x versions, undisclosed requests may bypass iControl REST authentication.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Heyder Andrade' # Metasploit module
+        ],
+        'References' => [
+          ['CVE', '2022-1388'],
+          ['URL', 'https://support.f5.com/csp/article/K23605346']
```

Comment:
You might want to add/check out https://twitter.com/jas502n/status/1523611433938059265 :)

Reply:
Link to analysis and/or PoC used, please.